recipes/pyoptsparse/meta.yaml

```
@@ -0,0 +1,82 @@
+{% set name = "pyoptsparse" %}
+{% set version = "2.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/mdolab/pyoptsparse/archive/refs/tags/v{{ version }}.tar.gz"
+  sha256: 4f28eefa4eb5724f3d0a2b6118c45aedee24a7dea72a6672a49feb77220ecf32
+  patches:
+    - patch-IPOPT-setup-for-conda.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+
+  host:
+    - python
+    - pip
+    - numpy
+    - swig
+    - ipopt 3.13.*
```

Constraining this will probably cause more harm than good in the long run
```suggestion
    - ipopt
```

---

I'm not familiar with conda at all, but since we are linking to the ipopt library, would there be problems if the conda package was built with say 3.13 but the user has 3.12 installed?

---

That won't happen. `run_exports` will make sure the correct run dependency is added.